lte/gateway/c/session_manager/SessionStore.cpp

```diff
@@ -185,6 +190,24 @@ bool SessionStore::update_sessions(const SessionUpdate& update_criteria) {
   return store_client_->write_sessions(std::move(session_map));
 }
 
+void SessionStore::initialize_metering_counter() {
+  auto session_map = store_client_->read_all_sessions();
+  for (auto& sessions_by_imsi : session_map) {
+    const std::string imsi = sessions_by_imsi.first;
+    for (auto& session : sessions_by_imsi.second) {
+      const std::string session_id = session->get_session_id();
+      auto total_usage             = session->get_total_credit_usage();
+      MLOG(MINFO) << "Initializing metering metrics with on startup for "
```

Comment:
is this sentence correct?

Reply:
Isn't this too verbose? or is it just temporal?

Reply:
will fix sentence haha :) It's a little verbose, but it's only printed on startup. Might bump it down to debug.